pmm-app/tests/pages/pmmSettingsPage.js

```diff
@@ -116,7 +116,23 @@ module.exports = {
     tooltipSelector: '.popper__background',
     validationMessage: 'span.error-message',
     tabsSection: '[data-qa="settings-tabs"]',
-    tabContent: '[data-qa="settings-tab-content"]'
+    tabContent: '[data-qa="settings-tab-content"]',
+    perconaPlatformLink: "//li[contains(text(), 'Percona Platform')]",
+    signUpEmail: '[data-qa="sign-up-email-input"]',
+    signUpPassword: '[data-qa="sign-up-password-input"]',
+    signUpAgreementLabel: '[data-qa="sign-up-agreement-checkbox-label"]',
+    signUpButton: '[data-qa="sign-up-submit-button"]',
+    singInToSignUpButton: '[data-qa="sign-in-to-sign-up-button"]',
+    signUpBackToLogin: '[data-qa="sign-up-to-sign-in-button"]',
+    downloadServerDiagnostics: '[data-qa="diagnostics-button"]',
+    diagnosticsTooltip:
```

Comment:
This text is already available in diagnosticText, please check the settingsPage.